rosdep/base.yaml

```diff
@@ -3463,6 +3463,10 @@ pocketsphinx:
     xenial: [pocketsphinx]
     yakkety: [pocketsphinx]
     zesty: [pocketsphinx]
+polyclipping:
```

Comment:
please use `-dev` in the name to avoid confusion with the non dev version `libpolyclipping`